packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -268,7 +268,7 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
       agentContext.config.logger.trace(
         `Submitting get credential definition request for credential definition '${credentialDefinitionId}' to ledger '${pool.indyNamespace}'`
       )
-      const response = await pool.submitRequest(request)
+      const response: GetCredentialDefinitionResponse = await pool.submitRequest(request)
```

Comment:
Is this needed? The response type should automatically be inferred from the request

Reply:
It was not automatically inferring the type for me when I was developing, not sure if it was an issue with my environment or not.